tests/integration_tests/tests/utils.py

```diff
@@ -35,18 +39,48 @@ def get_cfy():
     return utils.get_cfy()
 
 
-def upload_mock_plugin(package_name, package_version):
+def upload_mock_plugin(package_name, package_version, plugin_path='plugins'):
     client = create_rest_client()
-    temp_file_path = utils._create_mock_wagon(
+    temp_file_path = _create_mock_wagon(
             package_name,
-            package_version)
-    yaml_path = get_resource('plugins/plugin.yaml')
+            package_version,
+            )
+    # path relative to resources folder
+    yaml_path = get_resource(plugin_path+"/plugin.yaml")
```

Comment:
Try to use instead
`yaml_path = get_resource(os.path.join(plugin_path, 'plugin.yaml'))`